src/themes/default/store/homepage.ts

```diff
@@ -19,16 +20,31 @@ export const homepageStore = {
         { products: newProductsResult.items
         }, { root: true })
       commit('SET_NEW_COLLECTION', configuredProducts)
+    },
+    async fetchBestsellers ({ commit, dispatch }) {
```

Comment:
as we established some time ago, when we fetch and change state, then methods should be called `load...` when just fetching and not mutating state, then `fetch...`
```suggestion
    async loadBestsellers ({ commit, dispatch }) {
```